table/tables/tables_test.go

```diff
@@ -338,7 +338,9 @@ PARTITION BY RANGE ( id ) (
 		PARTITION p3 VALUES LESS THAN (21)
 )`
 
-	_, err := ts.se.Execute(context.Background(), createTable1)
+	_, err := ts.se.Execute(context.Background(), "set @@session.tidb_enable_table_partition=1")
```

Comment:
So we don't need to set `tbInfo.Partition.Enable = true`?

Reply:
Done.